apps/blockchain/starknet/src/tests/bridge_t.cairo

```diff
@@ -694,81 +694,149 @@ mod tests {
         let collection1 = starknet::contract_address_const::<'collection1'>();
         let collection2 = starknet::contract_address_const::<'collection2'>();
         let collection3 = starknet::contract_address_const::<'collection3'>();
-
+        let collection4 = starknet::contract_address_const::<'collection4'>();
+        let collection5 = starknet::contract_address_const::<'collection5'>();
+  
+        let mut spy = spy_events(SpyOn::One(bridge_address));
         start_prank(CheatTarget::One(bridge_address), BRIDGE_ADMIN);
         bridge.white_list_collection(collection1, true);
         bridge.white_list_collection(collection2, true);
         bridge.white_list_collection(collection3, true);
+        bridge.white_list_collection(collection4, true);
+        bridge.white_list_collection(collection5, true);
         stop_prank(CheatTarget::One(bridge_address));
         
         let white_listed = bridge.get_white_listed_collections();
-        assert_eq!(white_listed.len(), 3, "White list shall contain 3 elements");
+        assert_eq!(white_listed.len(), 5, "White list shall contain 5 elements");
         assert_eq!(*white_listed.at(0), collection1, "Wrong collection address in white list");
         assert_eq!(*white_listed.at(1), collection2, "Wrong collection address in white list");
         assert_eq!(*white_listed.at(2), collection3, "Wrong collection address in white list");
+        assert_eq!(*white_listed.at(3), collection4, "Wrong collection address in white list");
+        assert_eq!(*white_listed.at(4), collection5, "Wrong collection address in white list");
         assert!(bridge.is_white_listed(collection1), "Collection1 should be whitelisted");
         assert!(bridge.is_white_listed(collection2), "Collection1 should be whitelisted");
         assert!(bridge.is_white_listed(collection3), "Collection1 should be whitelisted");
+        assert!(bridge.is_white_listed(collection4), "Collection1 should be whitelisted");
+        assert!(bridge.is_white_listed(collection5), "Collection1 should be whitelisted");
 
```

Comment:
Could you please move this line 
```cairo
let mut spy = spy_events(SpyOn::One(bridge_address));
```
just before
```cairo
        start_prank(CheatTarget::One(bridge_address), BRIDGE_ADMIN);
        bridge.white_list_collection(collection3, false);
        stop_prank(CheatTarget::One(bridge_address));
```
to ensure that `assert_emitted` only check events from:
```cairo
bridge.white_list_collection(collection3, false);
```

Reply:
alright @ptisserand